Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -270,6 +269,9 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     m_extensionController = nil;
     m_contentScriptWorld = nullptr;
 
+    m_declarativeNetRequestDynamicRulesStore = nullptr;
+    m_declarativeNetRequestSessionRulesStore = nullptr;
+    m_declarativeNetRequestRuleStore = nullptr;
```

Comment:
These might make sense to go in `invalidateStorage()` which @kiaraarose just added.

Reply:
All of the DNR variables, or just these three?

Reply:
I ended up making a new method and put all of the state clearing there.